docs/spec.md

```diff
@@ -667,7 +667,8 @@ When multiple annotations are present, the Ion processor will maintain
 their order. Duplicate annotation symbols are allowed but discouraged.
 
 In the text format, type annotations are denoted by a non-null symbol
-token and double-colons preceding any content:
+token and double-colons preceding any value. Multiple annotations on the same
+value are separated by double-colons:
```

Comment:
I notice the example list just below this only includes a single example of a value with multiple annotations and it's a struct field (which is already a bit confusing for Ion newcomers).

Could you add a plain value with multiple annotations to the examples? (`degrees::'celsius'::100`?)